bazel/external/v8.patch

```diff
@@ -1,6 +1,7 @@
 # 1. Disable pointer compression (limits the maximum number of WasmVMs).
 # 2. Don't expose Wasm C API (only Wasm C++ API).
 # 3. Fix gcc build error by disabling nonnull warning.
+# 4. Allow compiling v8 on macOS 10.15 to 13.0. TODO(dio): Will remove this patch when https://bugs.chromium.org/p/v8/issues/detail?id=13428 is fixed.
```

Comment:
FYI, this was fixed in [v8@de4e4926](https://chromium.googlesource.com/v8/v8/+/de4e49260f908ca31ee64794c89b04cb004554be) (2023-02-02), so V8 update is also in order.

Reply:
Yup, working on it